docs/en/sql-reference/statements/create/dictionary.md

```diff
@@ -82,6 +82,28 @@ LIFETIME(MIN 0 MAX 1000)
 LAYOUT(FLAT())
 ```
 
+:::note
+When using the SQL console in [ClickHouse Cloud](https://clickhouse.com), you must specify a user (`default` or any other user) and password when creating a dictionary.
+:::note
+
+```sql
```

Comment:
```suggestion
```sql
CREATE USER IF NOT EXISTS clickhouse_admin
IDENTIFIED WITH sha256_password BY 'passworD43$x';

GRANT default_role TO clickhouse_admin;

```